src/apps/stable/features/playback/utils/playbackSubscriber.ts

```diff
@@ -11,13 +11,15 @@ import Events, { type Event } from 'utils/events';
 import { PlaybackManagerEvent } from '../constants/playbackManagerEvent';
 import { PlayerEvent } from '../constants/playerEvent';
 import type { ManagedPlayerStopInfo, MovedItem, PlayerError, PlayerErrorCode, PlayerStopInfo, RemovedItems } from '../types/callbacks';
+import { MediaSegmentDto } from '@jellyfin/sdk/lib/generated-client';
```

Comment:
We should import from `...generated-client/models/media-segment-dto`

Reply:
Fixed in `skipsegment` as well.